### PR TITLE
feat: Dashboard loading placeholder and fix user avatar

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

File: webapp/src/views/projects/dashboard/DashboardView.tsx [hunk 1]
```diff
@@ -98,27 +147,27 @@ export const DashboardView = () => {
...
-          )}
-          <Box gridArea="activityList">
-            <ActivityList />
-          </Box>
-          <Box gridArea="activityChart">
-            <DailyActivityChart />
-          </Box>
-        </StyledContainer>
+        {anythingLoading ? (
+          <EmptyListMessage loading={true} />
```
Comment (JanCizmar) [inline webapp/src/views/projects/dashboard/DashboardView.tsx:151]: Base view has also loading props, doesn't it?

**Review by JanCizmar — APPROVED** [review]
Its fix, not a feature.

File: webapp/src/views/projects/dashboard/DashboardView.tsx (hunk 1 above)
Comment (stepan662) [reply webapp/src/views/projects/dashboard/DashboardView.tsx:151]: Yes, but that does loading at the top

Comment (JanCizmar) [reply webapp/src/views/projects/dashboard/DashboardView.tsx:151]: Oki doke.

**Pull request merged**